integration/containerd/confidential/agent_image.bats

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bats
+# Copyright (c) 2022 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/lib.sh"
+
+setup() {
+	sandbox_name="kata-cc-busybox-sandbox"
+	crictl_delete_cc_pod_if_exists "$sandbox_name"
+
+	echo "Prepare containerd for Confidential Container"
+	SAVED_CONTAINERD_CONF_FILE="/etc/containerd/config.toml.$$"
+	configure_cc_containerd "$SAVED_CONTAINERD_CONF_FILE"
+
+	echo "Reconfigure Kata Containers"
+	switch_image_service_offload on
+}
+
+@test "[cc][agent] Test can pull an unencrypted image inside the guest" {
+	local pod_config="${FIXTURES_DIR}/pod-config.yaml"
+	local container_config="${FIXTURES_DIR}/container-config.yaml"
+
+	# TODO: add a disable_full_debug to revert the changes on teardown.
+	enable_full_debug
```

Comment:
There might be an argument that the debug config shouldn't be used in the pipeline, especially as it introduces potential security holes? It depends if we need it to grab required logs long term?

Reply:
Good point.

Maybe I should break this `enable_full_debug` into small enable_FOO_debug methods. For this test in particular, while running on CI (i.e. `CI=true`) we only want the agent logs for debugging in case the test fails. However, if you as a developer wants to debug a problem locally, enabling other debug like runtime is really appreciated.

I will refactor that code and send a v2.